.gitignore

```diff
@@ -178,6 +172,16 @@ local.properties
 .scala_dependencies
 .worksheet
 
+# Project files
+**/.classpath
+**/.project
+
+# m2e-code-quality for Eclipse CS Checkstyle / PMD / FindBugs Plug-Ins
+**/.checkstyle
+**/.pmd
+**/.pmdruleset.xml
+**/.fbExcludeFilterFile
+
 ### Kotlin template
 # Compiled class file
 
```

Comment:
If of any interest, here are the eclipse non-ignored files I spotted before the PR:
`.project`
`codec-framework/.checkstyle`
`codec-framework/.classpath`
`codec-framework/.project`
`connector-examples/.project`
`crypto-conditions/.checkstyle`
`crypto-conditions/.classpath`
`crypto-conditions/.project`
`dev-ops/.classpath`
`dev-ops/.project`
`ilp-annotations/.checkstyle`
`ilp-annotations/.classpath`
`ilp-annotations/.project`
`ilp-core/.checkstyle`
`ilp-core/.classpath`
`ilp-core/.project`
`ilp-core-codecs/.checkstyle`
`ilp-core-codecs/.classpath`
`ilp-core-codecs/.project`
`jackson-datatypes/.project`
`jackson-datatypes/jackson-datatype-cryptoconditions/.checkstyle`
`jackson-datatypes/jackson-datatype-cryptoconditions/.classpath`
`jackson-datatypes/jackson-datatype-cryptoconditions/.project`
`jackson-datatypes/jackson-datatype-interledger-address/.checkstyle`
`jackson-datatypes/jackson-datatype-interledger-address/.classpath`
`jackson-datatypes/jackson-datatype-interledger-address/.project`
`jackson-datatypes/jackson-datatype-interledger/.checkstyle`
`jackson-datatypes/jackson-datatype-interledger/.classpath`
`jackson-datatypes/jackson-datatype-interledger/.project`